grails-app/conf/BootStrap.groovy

```diff
@@ -157,7 +170,11 @@ class BootStrap {
                     backgroundColor : location?.bgColor,
                     zoneName : location?.zone?.name,
                     zoneId : location?.zone?.id,
-                    active : location?.active
+                    active : location?.active,
+                    organization : location?.organization,
+                    manager: location?.manager,
+                    address: location?.address,
+                    supportedActivities: location.supportedActivities ?: location.locationType?.supportedActivities,
```

Comment:
@jmiranda I am moving a question from previous pr to here:
> @jmiranda I am wondering if we want to include supportedActivities in this marshaller or if we want to keep it simpler and add a custom toJson mapper that will include these related objects like address and supportedActivities. What do you think about it?

For now, I am gonna merge it as-is and we can add a separate mapper later if needed.